api/include/opentelemetry/context/runtime_context.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "opentelemetry/context/context.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace context
+{
+// Provides a wrapper for propagating the context object globally. In order
+// to use either the threadlocal_context.h file must be included or another
+// implementation which must be derived from the RuntimeContext can be
+// provided.
+class RuntimeContext
+{
+public:
+  class Token
+  {
+  public:
+    bool operator==(const Context &other) { return context_ == other; }
+
+    ~Token() { Detach(*this); }
+
+  private:
+    friend class RuntimeContext;
+
+    // A constructor that sets the token's Context object to the
+    // one that was passed in.
+    Token(Context context) { context_ = context; }
+
+    Token() = default;
+
+    Context context_;
+
+    // Returns the stored context object.
+    Context GetContext() { return context_; }
```

Comment:
This method isn't used anywhere. What would it be needed for?

Reply:
It was initially used to get the context for comparison, but now that the comparison operator compares directly with contexts it is unnecessary, I will remove it.